src/UI/Components/Sidebar.php

```diff
@@ -346,7 +346,7 @@ public function getOutput()
                             } elseif (count($unpinnedMessages) == 2) {
                                 $height = 197;
                             }
-                            echo "<table id='messageWallWidget' style='width: 100%; height: ".$height."px; border: 1px solid grey; padding: 6px; background-color: #eeeeee'>";
+                            echo "<table id='messageWallWidget' style='width: w-auto; height: ".$height."px; border: 1px solid grey; padding: 6px; background-color: #eeeeee'>";
```

Comment:
w-auto is a class name rather than a CSS value, so it won't work here in this case.

```suggestion
                            echo "<table id='messageWallWidget' style='width: 100%; height: ".$height."px; border: 1px solid grey; padding: 6px; background-color: #eeeeee'>";
```